### PR TITLE
Feature/multiple choice

### PR DESCRIPTION
**Pull request by DannyCrews**
Merge multiple choice question into develop.

### CODE REVIEW

**Comment by DannyCrews** [comment]
/dist files are now not being tracked

File: src/blocks/bulb-tf/controls.js [hunk 1]
```diff
@@ -1,10 +1,5 @@
-import classnames from 'classnames';
-import icons from './icons';
-
-const { __ } = wp.i18n;
 const { Component } = wp.element;
 const { AlignmentToolbar, BlockControls, BlockAlignmentToolbar } = wp.editor;
-const { Toolbar, Button, Tooltip } = wp.components;
```
Comment (carlosesilva) [inline src/blocks/bulb-tf/controls.js]: No need to change this PR but for future purposes, these changes to TF could probably have been done on a separate pull request as they are not really related to topic of this PR ("the addition of the multiple choice question block").

File: src/blocks/bulb-mc/index.js [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+import './styles/editor.scss';
+import Answers from '../../components/Answers';
+
+import Inspector from './inspector';
+import Controls from './controls';
+
+const { __ } = wp.i18n;
+const { registerBlockType } = wp.blocks;
+const { RichText } = wp.editor;
+const { Fragment } = wp.element;
```
[the remark this answers is not in the document]
Comment (DannyCrews) [reply src/blocks/bulb-mc/index.js]: I'm playing around with Fragment a bit. I've actually put it back in locally.

File: src/blocks/bulb-mc/styles/style.scss [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+// 	h2 {
+// 		color: $dark;
+// 		margin: 1rem 0;
+// 	}
+
+// 	li {
+// 		list-style-position: inside;
+// 	}
+
+// }
```
[the remark this answers is not in the document]
Comment (DannyCrews) [reply src/blocks/bulb-mc/styles/style.scss]: I would agree with that for a production branch. I'm leaving commented code in during this phase of development for ease of reference.

File: src/blocks/bulb-mc/icons.js [hunk 4]
```diff
@@ -0,0 +1,43 @@
+const icons = {};
```
[the remark this answers is not in the document]
Comment (DannyCrews) [reply src/blocks/bulb-mc/icons.js]: Leaving it in as a model for setting up custom icons. I created a Trello card for that.

File: src/js/master.js [hunk 5]
```diff
@@ -76,6 +76,8 @@ class BULBQuestion {
 	slickQuizParseData() {
 		switch ( this.rawData.type ) {
 			case 'true-false':
+                return this.slickQuizParseDataTrueFalse();
+            case 'multiple-choice':
```
[the remark this answers is not in the document]
Comment (DannyCrews) [reply src/js/master.js]: fixed

File: src/blocks/bulb-mc/index.js [hunk 6]
```diff
@@ -0,0 +1,127 @@
...
+		};
+		const onChangeAnswers = newAnswers => {
+			setAttributes( {
+				answers: newAnswers,
+			} );
+		};
+
+		return (
+			<div className="quizDescription">
+					<Inspector { ...{ setAttributes, ...props } } />
```
[the remark this answers is not in the document]
Comment (DannyCrews) [reply src/blocks/bulb-mc/index.js]: fixed

**Pull request merged**